cmd/kubeapps-apis/proto/kubeappsapis/core/packages/v1alpha1/repositories.proto

```diff
@@ -222,6 +230,22 @@ message DockerCredentials {
   string email = 4;
 }
 
+// SshCredentials
+//
+message SshCredentials {
+  // private key
+  string private_key = 1;
+  // known hosts.
+  string known_hosts = 2;
```

Comment:
Should this be `repeated string known_hosts` to capture a list of known hosts? Fine either way, guessing you've talked about this in the meeting.

Reply:
this is what i thought originally, but after reading about what known hosts means, and how it is used in carvel, it seems the expectation is more of it being the content of the known_hosts file that you would find in the .ssh/ folder